packages/caver-klay/caver-klay-accounts/src/index.js

```diff
@@ -245,16 +280,22 @@ Accounts.prototype.signTransaction = function signTransaction(tx, privateKey, ca
 
         const messageHash = Hash.keccak256(rlpEncoded)
 
-        const signature = Account.makeSigner(Nat.toNumber(transaction.chainId || "0x1") * 2 + 35)(messageHash, privateKey)
-        const [v, r, s] = Account.decodeSignature(signature).map(sig => utils.makeEven(utils.trimLeadingZero(sig)))
+        let signatrues = []
```

Comment:
```suggestion
        let signatures = []
```

Reply:
oh.... there are many mistypo ;-;.. i will edit follow logic with this.